docs/start-stop.rst

```diff
@@ -117,3 +117,29 @@ with a README documenting the installation instructions, is available
 within the source repository at ``util/scripts/lsfcsm``.
 
 Support for the SLURM resource manager is under development.
+
+-----------------------------------------------
+  Stage-in and Stage-out Manifest File Format
+-----------------------------------------------
+
+The manifest file contains one or more file copy requests.
```

Comment:
Just curious, do things work as expected if a user gives a file with zero entries?  If so, we could say "zero or more" here.